### DISCUSSION TITLE
Tailwind Support

### DISCUSSION THREAD

**Discussion by skino2019** [spatie/laravel-google-fonts#18] [Q&A]
HI All! im just wondering if anyone has got this working with TailwindCSS or not?

this package would be spot on if i can get it working with tailwind.

**Comment by ju5t** [comment]
You can add the `@googlefonts` directive to your template and follow the manual one Tailwind CSS, see https://tailwindcss.com/docs/font-family#using-custom-values. 

We imported the fonts before and switching to `@googlefonts` was a drop-in replacement for us.

**Reply by Meraik** [reply]
Can you please elaborate?

**Reply by ju5t** [reply]
There isn't much to elaborate? We followed the instructions from this package, added `@googlefonts` to our blade view and were already following the instructions on https://tailwindcss.com/docs/font-family#using-custom-values. That's it really.
